docs/installation/updating-with-npm.md

```diff
@@ -0,0 +1,34 @@
+# Updating GOV.UK Frontend using node package manager (NPM)
+
+You can update with node package manager (NPM) if you [originally installed
```

Comment:
What's the reasoning behind the "NPM" abbreviation?
The product is branded as "npm" (https://www.npmjs.com/) so we could probably do away with the all caps abbreviation.
Also since the product is branded as "npm", I'm wondering whether we should have it as "node package manager, npm" or similar 🤔

Reply:
Since the audience for this guidance is web developers, the majority of them will not know npm by it's extended name.

The tool itself is nearly always referred to as `npm`, for example when you use it `npm install`

You can see on Google Trends when you compare the two, there is nearly no activity for the extended version. https://trends.google.com/trends/explore?geo=GB&q=npm,node%20package%20manager

So in this case I would only expand it once and then always use 'npm' for the rest of the guidance.

It's also worth nothing that npm should be lowercase not uppercase.

Reply:
Thanks both. Sounds like we're in agreement - expand the first time (keeping to style), use lowercase 'npm' (as that's the name). I'll fix that up.

@bravegrape, the brackets/comma question is an interesting one! I think our style is still brackets even if the acronym is the known name / product name. A similar-ish GOV.UK example is https://www.gov.uk/make-a-sorn - most people know it as SORN and definitely not 'Statutory Off Road Notification', but we keep the brackets. What do you think?

Reply:
Sounds good to me - let's follow the established pattern 👍